x-pack/plugins/cases/server/connectors/servicenow/sir_format.ts

```diff
@@ -45,20 +45,25 @@ export const format: ServiceNowSIRFormat = (theCase, alerts) => {
 
   if (fieldsToAdd.length > 0) {
     sirFields = alerts.reduce<Record<SirFieldKey, string[]>>((acc, alert) => {
+      let temp = {};
       fieldsToAdd.forEach((alertField) => {
         const field = get(alertFieldMapping[alertField].alertPath, alert);
+
         if (field && !manageDuplicate[alertFieldMapping[alertField].sirFieldKey].has(field)) {
           manageDuplicate[alertFieldMapping[alertField].sirFieldKey].add(field);
-          acc = {
```

Comment:
Fix for the linting rule prohibiting the direct assignment of function arguments